egs/babel_universal_am/s5/local/chain/run_ivector_common.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
```

Comment:
I think you should upgrade the script to the newer one provided in babel -- it's much faster and simpler. Not sure what are the modifications in this one.

Reply:
That's great to hear. I would have just used the newer script, but I was afraid it might not be compatible for some reason, so I just copied the version I used for now. I'll check more thoroughly through the new version and update this script if everything looks good.